src/In.ProjectEKA.HipService/DataFlow/DataFlow.cs

```diff
@@ -0,0 +1,37 @@
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    using System;
+    using System.Threading.Tasks;
+    using HipLibrary.Patient.Model.Response;
+    
+    public class DataFlow: IDataFlow
+    {
+        private readonly IDataFlowRepository dataFlowRepository;
+
+        public DataFlow(IDataFlowRepository dataFlowRepository)
+        {
+            this.dataFlowRepository = dataFlowRepository;
+        }
+        
+        public async Task<Tuple<HealthInformationResponse, ErrorResponse>> HealthInformationRequestFor(HealthInformationRequest request)
+        {
+            if (!ValidateConsentArtefact(request.Consent))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4941971)